src/main/resources/db/migration/common/V1_210__insert_transcription_status_data.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO darts.transcription_status VALUES(1, 'Requested');
+INSERT INTO darts.transcription_status VALUES(2, 'Awaiting Authorisation');
+INSERT INTO darts.transcription_status VALUES(3, 'With Transcriber');
+INSERT INTO darts.transcription_status VALUES(4, 'Complete');
+INSERT INTO darts.transcription_status VALUES(5, 'Rejected');
+INSERT INTO darts.transcription_status VALUES(6, 'Closed');
```

Comment:
In answer to my own question, I see the LLD State Model shows 

"Closed" - if a request has open for more than 30 days it will be closed by the [close transcription requests ](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1689786831#TaskautomationLLD-Closetranscriptionrequest(Previouslyknownas)task

@karen-hedges Please can you liase with DBA to add the extra status to their standing_data script?